wiki/deploy.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wiki
+  namespace: default
+  labels:
+    app: wiki
+  annotations:
+    fluxcd.io/automated: 'true'
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: wiki
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      name: wiki
+      labels:
+        app: wiki
+    spec:
+      containers:
+      - name: wiki
+        image: requarks/wiki:2
```

Comment:
Je kan hier de volledige versie neerzetten, dan kan Flux die updaten naar een nieuwe versie wanneer beschikbaar. Voordeel is dat het dan duidelijk is wanneer er ge-update wordt, ipv op een willekeurig tijdstip bij een container restart. Flux heeft ook semver filtering als je niet zomaar een major update krijgt.

Reply:
Oh nice dan pas ik dat aan